ui/components/Text.tsx

```diff
@@ -42,6 +42,9 @@ const Text = styled.span<TextProps>`
 
   ${(props) => props.noWrap && "white-space: nowrap;"};
```

Comment:
We are not adding semicolons in styled components styling we can just remove it too for consistency.